buildinfo_test.go

```diff
@@ -881,6 +885,10 @@ func validateBuildAddDepsBuildInfo(t *testing.T, buildInfoTestParams buildAddDep
 		"%s test with the command: \nrt bad %s  \nexpected to have the following dependencies: \n%s \nbut has: \n%s",
 		buildInfoTestParams.description, buildInfoTestParams.commandArgs, buildInfoTestParams.expectedDependencies, dependenciesToPrintableArray(buildInfo.Modules[0].Dependencies))
 
+	assert.Equalf(t, len(buildInfoTestParams.expectedModule), len(buildInfo.Modules[0].Id),
```

Comment:
It is wrong to compare the size of the module names. The actual module names should be compared.